src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -48,7 +47,10 @@ const AnnotationActivity = ({
     onSelect = noop,
 }: Props) => {
     const [isEditing, setIsEditing] = React.useState(false);
+    const [isMenuOpen, setIsMenuOpen] = React.useState(false);
     const { created_at, created_by, description, error, file_version, id, isPending, permissions = {}, target } = item;
+    const isFileVersionUnavailable = file_version === null;
```

Comment:
Can we group all of the derived state together either before or after the handlers? The split is a little confusing right now.